### PR TITLE
Deployment/sort dependencies

### PR DESCRIPTION
**Pull request by hermann-p**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -101,6 +89,16 @@
...
     "recompose": "^0.30.0",
-    "redux": "^4.0.1"
+    "redux": "^4.0.1",
+    "rxjs": "^6.4.0",
+    "sass": "^1.17.2",
+    "storybook": "^1.0.0",
+    "superagent": "^4.0.0",
+    "node-sass": "^4.10.0"
+  },
+  "dependencies": {
```
Comment (Zwergal) [inline package.json]: The Server uses the package "serve-static". This should also be in dependencies?!

Comment (hermann-p) [reply package.json]: Did you test that? `serve-static` should be a `node` core module.  
`npm i && npm prune --production && npm run build && npm run start` seems to work fine.

Comment (Zwergal) [reply package.json]: Yes it is a dependency of express. But we now rely on a dependency from a dependency. What if express will drop serve-static in favour of another module? Our code will break.

If we don't want server-static as a direct dependency then in the grud-frontend-server.js file we should not require it directly but use `express.static` instead.

Comment (hermann-p) [reply package.json]: I checked the code - you're right; I thought we had used express' core module, but we use a transient dependency of express. Added that.

**Pull request merged**